src/components/MenuProductModal/Modal.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactElement } from 'react';
+import { ProductDto } from 'dtos/ProductDto';
+import { MenuProductModal } from './MenuProductModal';
+
```

Comment:
I don't think this component is necessary, just wrap the productModal in the conditional and delete this